book/09-git-and-other-scms/sections/client-bzr.asc

```diff
@@ -0,0 +1,102 @@
+==== Git and Bazaar
```

Comment:
Instead of just jumping in, could you include a small introductory paragraph or two about Bazaar? Take a look at [the Mercurial section](https://github.com/aollier/progit2/blob/c85bf5f170343702d63b1ebefc897e516c8694f9/book/09-git-and-other-scms/sections/client-hg.asc) for the kind of thing I'm thinking of.